website/docs/cloud-docs/api-docs/state-versions.mdx

```diff
@@ -164,6 +172,26 @@ curl \
 }
 ```
 
+## Upload State and JSON State
+
+-> **Note:** While you can upload state version content in the same request when creating a state version, we strongly recommend that you upload content separately.
+
+`PUT https://archivist.terraform.io/v1/object/<UNIQUE OBJECT ID>`
+
+**The URL is the `hosted-state-upload-url` or `hosted-json-state-upload-url` returned when you create a `state-version`. Once you upload state content, the URL is hidden on the resource and no longer available.**
```

Comment:
We don't bold for emphasis in the docs, we use italics. If this is important for folks to know and needs more emphasis, we could add a call out?
```suggestion
Terraform Cloud returns a `hosted-state-upload-url` or `hosted-json-state-upload-url` when you create a `state-version`. Once you upload state content, this URL is hidden on the resource and _no longer available_.
```